client-v2/src/routes/routes.tsx

```diff
@@ -0,0 +1,56 @@
+import { priorities } from '../constants/priorities';
+import { matchPath } from 'react-router';
+
+export const base = '/v2';
+
+const editionsApi = (path: string) => `/editions-api${path}`;
+
+/**
+ * Fronts edit
+ */
+
+interface FrontsEditParams {
+  priority: string;
+}
+
+const frontsEditPathProps = {
+  exact: true,
+  path: `/:priority(${Object.keys(priorities).join('|')})`
+};
+
+const matchFrontsEditPath = (path: string) =>
+  matchPath<FrontsEditParams>(path, frontsEditPathProps);
+
+/**
+ * Issue
+ */
+
+interface IssueParams {
+  // The `priority` in this path is actually the edition issueId, but is mapped to priority
+  // so as to perserve things like favourite fronts and open fronts
+  priority: string;
```

Comment:
We might want to ship this for demo, so 👍, but I'd love to sit down and solve this problem in such a way that this is unnecessary!

Reply:
I wholly agree